js/playgame.js

```diff
@@ -74,10 +76,28 @@ PlayGame.prototype.update = function () {
             ROUNDS_PLAYED++;
             this.roundSecondsElapsed = 0;
             this.lastRoundWasTie = false;
-            this.goatWhoWonLastRound = null;
-
+            
+             for (var i = 0; i < this.game.goats.length; i++) {
+                this.goatScores.push(this.game.goats[i]);
+            }
+            
+            function bubbleSortDescending(items) {
```

Comment:
I believe this bubble sort can be replaced with this:
`this.goatScores.sort(function (a, b) {
                return b.score - a.score;
            });`